client/src/main/com/sinch/sdk/domains/verification/VerificationService.java

```diff
@@ -9,6 +9,17 @@
  */
 public interface VerificationService {
 
+  /**
+   * Use application secret in place of unified configuration for authentication (see Sinch
+   * dashboard for details)
+   *
+   * @param key see <a href="https://dashboard.sinch.com/voice/apps">dashboard</a>
+   * @param secret see <a href="https://dashboard.sinch.com/voice/apps">dashboard</a>
+   * @return service instance for project
+   * @since 1.0
+   */
+  VerificationService useSecrets(String key, String secret);
```

Comment:
Defining this method at `VerificationService` level means it will have to be defined again in the `VoiceService` and the user will have to enter again the application secrets.
This is strong mapping with the split in several OAS documents and domains: from an infrastructure point of view, the applications (holding the credentials) are simply belonging to an organisation, and can be accessed by any domain that needs them.

Reply:
Exact.
Sinch dashboard is displaying twice this information (once from `Verification` tab, and once, again, onto `Voice` tab) and this is reflected here... even if information are shared.
So, if moved at an upper level (SinchClient configuration) this mean we will have
- unified credentials
- SMS credentials
- "something embedding Verification & Voice & ...)" credentials

So to rephrase: all credentials at same level: the main configuration